src/controllers/metadata.js

```diff
@@ -75,14 +76,14 @@ module.exports = (config) => {
   }
 
   try {
-    metadata.version = `five-bells@${require('../../package.json').version.split('.')[0]}`
```

Comment:
This was pulled out to the top of the file, because having a require in a string template made `istanbul` (code coverage tool) choke.